### PR TITLE
Truncate long file attachment names for comments

### PR DESCRIPTION
**Pull request by kmshin1397**
Screenshot 2020-10-02 122542

### CODE REVIEW

File: static/js/components/CommentList.jsx [hunk 1]
```diff
@@ -19,6 +19,19 @@ import UserAvatar from "./UserAvatar";
 dayjs.extend(relativeTime);
 dayjs.extend(utc);
 
+export const shortenFilename = (filename) => {
+  if (filename.length <= 15) {
+    return filename;
+  }
+  const extensionLength = filename.split(".", 2)[1].length;
```
Comment (dmitryduev) [inline static/js/components/CommentList.jsx:26]: What if there are no dots in the filename for some reason?

**Review by dmitryduev — CHANGES_REQUESTED** [review]
Looks great, one small thing probably needs addressing.

**Review by acrellin — APPROVED** [review]
Thanks for the quick fix!

File: static/js/components/CommentList.jsx (hunk 1 above)
Comment (kmshin1397) [reply static/js/components/CommentList.jsx]: Good point. Case implemented

**Review by dmitryduev — APPROVED** [review]
LGTM!

**Pull request merged**